src/platform-includes/source-context/java.mdx

```diff
@@ -269,6 +311,18 @@ First, create the source bundle containing your source files:
 sentry-cli debug-files bundle-jvm --output path/to/store/bundle --debug-id A_VALID_UUID path/to/source-code
 ```
 
+<PlatformSection supported={["android"]}>
```

Comment:
I think we could skip the section if we use the `--include-sources` flag I mentioned below.

Reply:
right, we don't need this one